optimum_benchmark/backends/pytorch.py

```diff
@@ -72,10 +73,88 @@ class PyTorchConfig(BackendConfig):
     disable_grad: bool = "${is_inference:${benchmark.name}}"  # type: ignore
     eval_mode: bool = "${is_inference:${benchmark.name}}"  # type: ignore
 
+    # training options
+    use_ddp: bool = False
+    ddp_config: Optional[Dict[str, Any]] = None
+
+    def __post_init__(self):
+        """
+        Here we perform checks and transformations on the config.
+        But we never modify the types of the config values.
+        """
+
+        CUDA_VISIBLE_DEVICES = os.environ.get("CUDA_VISIBLE_DEVICES", None)
+
+        if self.torch_compile:
+            self.torch_compile_kwargs = OmegaConf.merge(
+                self.torch_compile_kwargs
+                if self.torch_compile_kwargs is not None
+                else {},
+                DEFAULT_COMPILE_CONFIG,
+            )
+
+        if self.device_map is not None:
+            assert self.device_map in ["auto", "sequential"], (
```

Comment:
I think it can make sense to use `ValueError` here (and below as well) for readability.

Reply:
thanks, I changed the errors to be more meaningful.